packages/bot-web-ui/src/components/dashboard/dashboard.tsx

```diff
@@ -113,6 +114,18 @@ const Dashboard = ({
     };
 
     React.useEffect(() => {
+        if (active_tab === 1) {
+            if (is_mobile) {
+                initTrashCan(-10);
```

Comment:
@rupato-deriv @prince-deriv we don't need the trash can in mobile, since we are not allowing the user to edit/update the strategy on mobile view. c: @sandeep-deriv

Reply:
done ✅